engine/src/main/java/io/knotx/fragments/engine/FragmentsEngine.java

```diff
@@ -55,15 +54,12 @@ public FragmentsEngine(Vertx vertx) {
 
     return Flowable.just(fragments)
         .concatMap(Flowable::fromIterable)
-        .map(fecta -> {
-          Optional<Node> rootNode = fecta.getTask().getRootNode();
-          if (rootNode.isPresent()) {
-            return taskEngine
-                .start(fecta.getTask().getName(), rootNode.get(), fecta.getFragmentEventContext());
-          } else {
-            return Single.just(fecta.getFragmentEventContext().getFragmentEvent());
-          }
-        })
+        .map(fecta -> fecta
```

Comment:
please use more meaningful variable name. it's hard to deduct what `fecta` stands for